einnahmen.js

```diff
@@ -96,77 +22,85 @@ Highcharts.setOptions({
   }
 });
 
-var parents = [];
-for (var i = 0; i < data.length; i++) {
-  data[i]['events'] = {'click': displaytext};
-  parents.push(data[i]['parent']);
-}
-parents = Array.from(new Set(parents));
-var colorSeq = palette('tol-rainbow', parents.length);
-var colorMap = {};
-for (var i = 0; i < parents.length; i++) {
-  colorMap[parents[i]] = "#" + colorSeq[i];
-}
-for (var i = 0; i < data.length; i++) {
-  if (data[i]['parent'] == "0.0") {
-    data[i]['color'] = colorMap[data[i]["id"]];
-  } else if (data[i]['id'] == "0.0") {
-    data[i]['color'] = '#ededed';
-  }
-}
+fetch('einnahmen.json')
```

Comment:
There seems to be something going wrong.

Reply:
fetch() will not work on your local machine, but it does work on the server.